tests/features/bootstrap/FeatureContext.php

```diff
@@ -29,6 +35,229 @@ public function assertAccessDenied() {
   }
 
   /**
+   * Assert that certain fields are present on the page.
+   *
+   * @param string $fields
+   *    Fields.
+   *
+   * @throws \Exception
+   *   Thrown when an expected field is not present.
+   *
+   * @Then /^(?:|the following )fields should be present? "(?P<fields>[^"]*)"$/
+   */
+  public function assertFieldsPresent($fields) {
+    $fields = $this->explodeCommaSeparatedStepArgument($fields);
+    $not_found = [];
+    foreach ($fields as $field) {
+      $is_found = $this->getSession()->getPage()->find('named', ['field', $field]);
+      if (!$is_found) {
+        $not_found[] = $field;
+      }
+    }
+    if ($not_found) {
+      throw new \Exception("Field(s) expected, but not found: " . implode(', ', $not_found));
+    }
+  }
+
+  /**
+   * Assert that certain fields are not present on the page.
+   *
+   * @param string $fields
+   *    Fields.
+   *
+   * @throws \Exception
+   *   Thrown when a column name is incorrect.
+   *
+   * @Then /^(?:|the following )fields should not be present? "(?P<fields>[^"]*)"$/
+   */
+  public function assertFieldsNotPresent($fields) {
+    $fields = $this->explodeCommaSeparatedStepArgument($fields);
+    foreach ($fields as $field) {
+      $is_found = $this->getSession()->getPage()->find('named', ['field', $field]);
+      if ($is_found) {
+        throw new \Exception("Field should not be found, but is present: " . $field);
+      }
+    }
+  }
+
+  /**
+   * Assert that certain fields are present and visible on the page.
+   *
+   * @param string $fields
+   *    Fields.
+   *
+   * @throws \Exception
+   *   Thrown when an expected field is not present or is not visible.
+   *
+   * @Then /^(?:|the following )fields should be visible? "(?P<fields>[^"]*)"$/
+   */
+  public function assertFieldsVisible($fields) {
+    $fields = $this->explodeCommaSeparatedStepArgument($fields);
+    $not_found = [];
+    $not_visible = [];
+    foreach ($fields as $field) {
+      $element = $this->getSession()->getPage()->find('named', ['field', $field]);
+      if (!$element) {
+        $not_found[] = $field;
+        continue;
```

Comment:
It's done to have a full list of what's wrong in that request. Otherwise you fix one and than have to wait for the next to appear on CPHP run.